### PR TITLE
spline #261 - Producer REST model

### PR DESCRIPTION
**Pull request by wajda**

### CODE REVIEW

File: producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/ExecutionEvent.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.producer.rest.model
+
+case class ExecutionEvent(
+  planId: ExecutionPlanId,
+  timestamp: Long
```
Comment (leborgneromain) [inline producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/ExecutionEvent.scala]: Should we add the errors later ?

File: producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/operationModel.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  override val parentIds: Array[OperationId],
+  override val schema: Option[Schema]
+) extends OperationLike
+
+case class ReadOperation(
+  inputSources: Array[SourceUrl],
+  override val id: OperationId,
+  override val schema: Option[Schema] = None
+) extends OperationLike {
+  override val parentIds: Array[OperationId] = Array.empty
```
Comment (leborgneromain) [inline producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/operationModel.scala]: Why this is initialized in Read Operation and not the others ?

Comment (wajda) [reply producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/operationModel.scala]: That's terminology confusion. I've renamed it to `childIds'

File: producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/ExecutionEvent.scala (hunk 1 above)
Comment (wajda) [reply producer-rest-model/src/main/scala/za/co/absa/spline/producer/rest/model/ExecutionEvent.scala]: added

**Pull request merged**